pkg/start/start.go

```diff
@@ -244,3 +260,44 @@ func makeBootstrapSuccessEvent(ns, name string) *corev1.Event {
 	}
 	return event
 }
+
+
+func isSingleNodeTopology(assetDir string) (bool, error) {
```

Comment:
Consider renaming to 
```suggestion
func isSingleNodeControlPlane(assetDir string) (bool, error) {
```